### PR TITLE
Adds custom banana slip sound

### PR DESCRIPTION
**Pull request by JohnGinnane**
Still have 5% to make the oof sound

:cl: JohnGinnane
soundadd: Banana peels now have their own slip sound
/:cl:

[why]: # More sounds mean more fun

### CODE REVIEW

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -198,7 +198,12 @@
 			C.log_message("<font color='orange'>Slipped[O ? " on the [O.name]" : ""][(lube&SLIDE)? " (LUBE)" : ""]!</font>", INDIVIDUAL_ATTACK_LOG)
 		if(!(lube&SLIDE_ICE))
 			if(prob(95))
-				playsound(C.loc, 'sound/misc/slip.ogg', 50, 1, -3)
+				// Hippie Start - custom sounds for banana
+				if (istype(O, /obj/item/grown/bananapeel))
```
Comment (ghost) [inline code/game/turfs/open.dm]: Add a `custom_slip_sound` var on objects, if it's set use that, otherwise use slip sound

**Review by ghost — DISMISSED** [review]

File: code/game/turfs/open.dm (hunk 1 above)
Comment (JohnGinnane) [reply code/game/turfs/open.dm]: Gonna wait for my other PR to be merged before that, as I'll add it to the /obj/item class as well

**Comment by JohnGinnane** [comment]
@HippieStation/head thoughts??

**Comment by bardofrage** [comment]
Why not

**Comment by ghost** [comment]
after a long period of serious thought I have decided to +1 this

**Comment by Spankmaster1066** [comment]
Why isn't this in yet +1

**Comment by KayeArray** [comment]
+1

**Comment by ghost** [comment]
yeah +1 oof

**Comment by JohnGinnane** [comment]
I'll work on this before merging to make it more.. accessible ))

**Comment by JohnGinnane** [comment]
Alrighty, I've refactored it so that *any* object could have an alternate slip sound, much like my alternate_screams. If there isn't one it'll use the default one. Tested this with water chempile and it still made the noise. Tested with normal soap and soap whose alternate slip sound list I added a sound to and it played.

@Chnkr can you re-review? I also think I have +3 now too

**Comment by JohnGinnane** [comment]
@Chnkr or anyone else

**Review by ghost — APPROVED** [review]

**Pull request merged**